automation/jleechanorg_pr_automation/tests/test_automation_safety_limits.py

```
@@ -442,8 +442,10 @@ def test_launchd_plist_includes_safety_wrapper(self):
 
     def run_automation_script(self):
         """Helper to run automation script"""
-        script_path = Path.home() / "projects" / "worktree_worker2" / "automation" / "simple_pr_batch.sh"
-        return subprocess.run([str(script_path)], check=False, capture_output=True, text=True)
+        import subprocess
```

Ruff (0.14.7)

`import` should be at the top-level of a file

(PLC0415)